regions/shapes/ellipse.py

```diff
@@ -130,7 +130,10 @@ def bounding_box(self):
         if dx1 > dx2:
             dx1, dx2 = dx2, dx1
 
-        t1 = np.arctan(self.height / tan_angle / self.width)
+        if 0 in (self.height, tan_angle, self.width):
```

Comment:
I don't think this will be robust to 0 vs 0.0, probably safer to use == for each in turn